core/new-gui/src/app/dashboard/component/login/login.component.ts

```diff
@@ -0,0 +1,133 @@
+import { Component, OnInit } from "@angular/core";
+import { UserService } from "../../../common/service/user/user.service";
+import { FormBuilder, FormControl, FormGroup, Validators } from "@angular/forms";
+import { isDefined } from "../../../common/util/predicate";
+import { filter } from "rxjs/operators";
+import { UntilDestroy, untilDestroyed } from "@ngneat/until-destroy";
+
+import { Router } from "@angular/router";
+
+@UntilDestroy()
+@Component({
+  selector: "texera-login",
+  templateUrl: "./login.component.html",
+  styleUrls: ["./login.component.scss"],
+})
+export class LoginComponent implements OnInit {
```

Comment:
it seems we are doing many things other than login on this component, for example we are showing some information for texera (blog, wiki), showing option for people to sign up, etc.

I think `LoginComponent` does not describe this page accurately. This component is in parallel with `WorkspaceComponent` and `DashboardComponent`, shall we name it as `LandingComponent`, where it has child components to do login, registration, information display, etc?

Reply:
Or `HomeComponent`?

Reply:
HomeComponent is good.